game/src/main/scala/io/aigar/game/GameThread.scala

```diff
@@ -65,7 +69,9 @@ class GameThread(scoreThread: ScoreThread) extends Runnable
         updateGames
       }
 
-      Thread.sleep(Game.MillisecondsPerTick)
+      Thread.sleep(math.max(0, round(Game.MillisecondsPerTick - (currentTime - previousTime) * Game.MillisecondsPerSecond)))
```

Comment:
If I understand correctly, this will make sure the `Thread` tick duration will be the same? We did something similar in the frontend, didn't we?